core/frontend/src/types/autopilot.ts

```diff
@@ -0,0 +1,62 @@
+export interface Firmware {
+    name: string
+    url: URL
+}
+
+export enum Vehicle {
+    Sub = 'Sub',
+    Rover = 'Rover',
+    Plane = 'Plane',
+    Copter = 'Copter',
+}
+
+export enum SITLFrame {
```

Comment:
Is this whole enum unused?

Reply:
Yup. I was adding this to allow SITL configuration, but I can add this later on the following PR.